periodic_tasks/reward_splitter/tasks.py

```diff
@@ -0,0 +1,224 @@
+import logging
+from typing import cast
+
+from eth_typing import ChecksumAddress, HexStr
+from hexbytes import HexBytes
+from web3 import Web3
+from web3.types import Wei
+
+from periodic_tasks.common.graph import get_multiple_harvest_params
+from periodic_tasks.common.networks import ZERO_CHECKSUM_ADDRESS
+from periodic_tasks.common.settings import EXECUTION_TRANSACTION_TIMEOUT
+from periodic_tasks.common.typings import HarvestParams
+from periodic_tasks.common.utils import to_chunks
+from periodic_tasks.exit.contracts import keeper_contract, multicall_contract
+from periodic_tasks.exit.typings import ExitRequest
+from periodic_tasks.reward_splitter.typings import RewardSplitter
+
+from . import settings
+from .clients import execution_client, graph_client
+from .contracts import RewardSplitterContract, RewardSplitterEncoder, VaultStateContract
+from .graph import graph_get_claimable_exit_requests, graph_get_reward_splitters
+
+logger = logging.getLogger(__name__)
+
+
+async def process_reward_splitters() -> None:
+    vaults = [Web3.to_checksum_address(v) for v in settings.VAULTS]
+    block = await execution_client.eth.get_block('finalized')
+
+    logger.info('Fetching reward splitters')
+    reward_splitters = await graph_get_reward_splitters(block_number=block['number'], vaults=vaults)
+
+    if not reward_splitters:
+        logger.info('No reward splitters found for given vaults')
+        return
+
+    vaults = [rs.vault for rs in reward_splitters]
+    vault_to_can_harvest_map = await _get_vault_to_can_harvest_map(vaults=vaults)
```

Comment:
You can fetch can harvest from the subgraph, so you can remove this call and modify get_multiple_harvest_params